app.css

```diff
@@ -8,8 +8,64 @@ html {
 
 body {
   margin: 0;
+  font-family: "Pavanam", sans-serif;
 }
 
 img {
   max-width: 100%;
 }
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.nav-bar {
+  padding: 1rem;
+  width: 100%;
+  background: #0ad9d9;
+  overflow: auto;
+}
+
+.nav-title {
+  margin: 2rem 1rem;
+  float: left;
+}
+  .nav-link {
+    font-size: 1.2rem;
+    margin: 1rem 1rem;
+    overflow: auto;
+  }
+
+  .nav-group__item {
+    float: right;
+    margin: 1rem;
+  }
+
+.container {
+  max-width: 65rem;
+  margin: 0 auto;
+}
+
+.grid {
+  margin: 1rem;
+  width: calc( 100% / 3 - 1rem - 1rem );
+}
+
+.profile__header-card {
+  background: #da310c;
+  position: relative;
+}
+
+.image {
+  border-radius: 100%;
+  margin: 1rem;
+  position: absolute;
+  left: -12%;
+  top: 14%;
```

Comment:
Instead of using random percentages, try using even values like (-)`100%`, (-)`50%`, or `0` and then using `transform` and `translate` to get it perfectly centered on the parent element.

This same trick can be used for the top "week" chip.

https://online.theironyard.com/paths/579/units/3469/lessons/13952